devel-requirements.txt

```diff
@@ -0,0 +1,178 @@
+alabaster==0.7.12
```

Comment:
You probably "freezed" your Python environment, don't you? That can happen as the file contains a lot of entries.

I would recommend to drastically reduce it to the bare minimum. We only need pytest for the time being.